thirdeye/thirdeye-frontend/app/styles/components/rootcause-anomaly.scss

```diff
@@ -61,4 +60,15 @@
   &__stats-value {
     font-size: 16px;
   }
+
+  &-toggle {
```

Comment:
Also a modifier, so this should be `--toggle`.

Reply:
fixed